Xwt.Gtk/Xwt.GtkBackend/ButtonBackend.cs

```diff
@@ -197,6 +199,37 @@ public void SetContent (string label, bool useMnemonic, ImageDescription image,
 				Widget.Label = null;
 		}
 		
+		FormattedText formattedText = null;
+		private void SetFormattedContent (FormattedText label, ContentPosition position)
+		{
+			if (label != null && label.Text.Length == 0)
```

Comment:
Don't we want to clear the content when label is null or it has no text?

This seems to exit when label is not null and the text is empty. Then when `labelWidget.ApplyFormattedText (formattedText)` is called, it's going to NRE inside, because formattedText is null.

Reply:
yep, good point